### PR TITLE
Debug blog post

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Comment by gep13** [comment]
@patriksvensson @devlead @mholo65 can I get you to have a look over this? Thanks!

**Comment by patriksvensson** [comment]
Looks good to me! :+1:

File: src/Cake.Web/App_Data/blog/2016-05-26-debug-cake-file.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+### break preprocessor directive
+
+In addition to directly setting breakpoints, it is also possible to place a `#break` pre-processor directive at the location where you want the debugger to break in your script.  At runtime, this `#break` directive will be replaced by `System.Diagnostics.Debugger.Break();` which will cause the debugger to stop at that location.
+
+![Using Break Directive](https://raw.githubusercontent.com/cake-build/website/master/src/Cake.Web/App_Data/images/debugging-cake-file/using_break_directive.png)
+
+### Gotchas!
+
+1. In order to allow debugging of Cake Scripts, Visual Studio must be running in an elevated mode.  If you attempt to debug a Cake script when not elevated, you will see the following message appear.
+<br/>![Elevated Permissions](https://raw.githubusercontent.com/cake-build/website/master/src/Cake.Web/App_Data/images/debugging-cake-file/elevated_permissions.png)
```
Comment (devlead) [inline src/Cake.Web/App_Data/blog/2016-05-26-debug-cake-file.md]: Is this true? I don't get that prompt. Running both Cake & Devenv as non admin.

Comment (gep13) [inline src/Cake.Web/App_Data/blog/2016-05-26-debug-cake-file.md]: Yip, happens on my machine every time.

Comment (mholo65) [inline src/Cake.Web/App_Data/blog/2016-05-26-debug-cake-file.md]: Doesn't happen to me either

Comment (devlead) [inline src/Cake.Web/App_Data/blog/2016-05-26-debug-cake-file.md]: Interesting, perhaps must is might then :)

Comment (patriksvensson) [inline src/Cake.Web/App_Data/blog/2016-05-26-debug-cake-file.md]: Now when I think about it, that has never appeared to me as well. Can we change it to something like "Visual Studio might need to run in elevated mode." or something similar?

File: src/Cake.Web/App_Data/blog/2016-05-26-debug-cake-file.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+1. Clone the Cake project from GitHub
+1. Open the Cake solution file `src/Cake.sln`
+1. Modify the project properties to specify the build.cake file that you would like to debug
+<br/>![Hit Breakpoint](https://raw.githubusercontent.com/cake-build/website/master/src/Cake.Web/App_Data/images/debugging-cake-file/project_properties.png)
+1. Start debugging by pushing F5, or clicking the green play button
+
+### break preprocessor directive
+
+In addition to directly setting breakpoints, it is also possible to place a `#break` pre-processor directive at the location where you want the debugger to break in your script.  At runtime, this `#break` directive will be replaced by `System.Diagnostics.Debugger.Break();` which will cause the debugger to stop at that location.
```
Comment (devlead) [inline src/Cake.Web/App_Data/blog/2016-05-26-debug-cake-file.md]: Maybe clarify if debugger is attached otherwise ignored?

**Comment by mholo65** [comment]
Maybe add that it doesn't work with mono. 

+move the elevated stuff to "known issues" and rephrase or remove it completely. There has to be something odd with your setup @gep13. Could you investigate this further?

Otherwise, great blog post!

**Comment by gep13** [comment]
>@mholo65 said...
+move the elevated stuff to "known issues" and rephrase or remove it completely

I have rephrased this.  Can you let me know what you think?  I am hesitant to remove it completey as it is happening on my machine, and may well happen on others as well, so would prefer to leave it in.

**Comment by gep13** [comment]
@patriksvensson @devlead @mholo65 I think I have incorporated all the feedback.

**Comment by patriksvensson** [comment]
@gep13 I think it looks good :+1:

**Comment by mholo65** [comment]
@gep13 Looks good to me also! :shipit:

**Comment by devlead** [comment]
@gep13 :shipit: :+1:

**Pull request merged**